README.md

```diff
@@ -53,11 +53,11 @@ This "Hello World!" example is built with the `0.9.0-SNAPSHOT` version of `finch
 
 ```scala
 import io.finch._
-import com.twitter.finagle.Http
```

Comment:
Finch is already on Finagle 6.30, which has [renamed `httpx` back to `http`](https://twitter.com/finagle/status/657233491340296192).